bpf/straceback-guess-bpf.h

```diff
@@ -25,7 +25,7 @@ struct guess_status_t {
 };
 
 #define MAX_TRACED_PROGRAMS 128
-#define MAX_POOLED_PROGRAMS 32
+#define MAX_POOLED_PROGRAMS 128
```

Comment:
Nothing here, but the message of the commit that introduces the changes talks about the diffference between MAX_POOLED_PROGRAM and MAX_POOLED_PROGRAMS instead of MAX_POOLED_PROGRAMS and MAX_TRACED_PROGRAMS.

Reply:
Also, I hope that this change is already reflected in the generated go-bindata file (the commit does not change the generated file).

Reply:
Thanks! commitmsg updated and I added a commit for the generated go-bindata.